sessionctx/variable/tidb_vars.go

```diff
@@ -599,6 +599,7 @@ var (
 	maxDDLReorgWorkerCount int32 = 128
 	ddlReorgBatchSize      int32 = DefTiDBDDLReorgBatchSize
 	ddlErrorCountlimit     int64 = DefTiDBDDLErrorCountLimit
+	ddlReorgRowFormat      int64 = DefTiDBRowFormatV1
```

Comment:
The default value should be DefTiDBRowFormatV2.

Reply:
good